source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -13,6 +15,11 @@ BioXASAppController::BioXASAppController(QObject *parent) :
 	userConfiguration_ = new BioXASUserConfiguration(this);
 	setDefaultUseLocalStorage(true);
 
+	// Database upgrades.
+
+	AMDbUpgrade *bioxas1pt1 = new BioXASDbUpgrade1Pt1("user", this);
+	appendDatabaseUpgrade(bioxas1pt1);
```

Comment:
Not a big deal for initial testing, but you will need to do this with the actions database too.  Due to old design constraints, the user and actions databases need to be the same (since they both can recall scan configurations).